RelationalAI/DefaultAccessTokenHandler.cs

```diff
@@ -32,34 +31,40 @@ namespace RelationalAI
     public class DefaultAccessTokenHandler : IAccessTokenHandler
```

Comment:
Needs handling for when filename not found -> INFO log msg, fallback to memory-based cache local ot the tokenhandler